tanium-threat-response/src/openbas_tanium_threat_response.py

```diff
@@ -128,6 +129,9 @@ def _extract_tree_commands(self, artifact, commands):
                 .replace(file_path, "")
```

Comment:
Maybe you can intilize comment as "" and after validate if if isinstance(artifact, dict)